### ISSUE TITLE
Editing items shown on map

### ISSUE THREAD

**Issue by tib1247gg** [smrtgVirtual/hillgrove-city#3]
I installed your map as it got updated to version 4.0. Unfortunately when the map loads, it loads fine but there is no AI vehicles or buses, and it shows some stuff that is not supposed to be shown. Some examples are it shows the arrows of where the AI vehicles go and also the buses, the passenger boarding point etc. Hopefully you are able to help check with this issue. If I did something wrong, please do tell me! I love this map a lot!

**Comment by smrtgVirtual** [comment]
Hi! Thank you for your feedback.

> ...it loads fine but there is no AI vehicles or buses,

 Will be mitigated through an update with the MAN A20 (given) as the AI buses.

Needs clarifications:

>...and it shows some stuff that is not supposed to be shown. Some examples are it shows the arrows of where the AI vehicles go and also the buses, the passenger boarding point etc.

Did you mean that the route hints continued to appear despite not toggling it on?

~G

**Comment by tib1247gg** [comment]
Yes correct. When I ask someone what happened, they say I was in editing mode but I wasn't

**Comment by smrtgVirtual** [comment]
> Yes correct. When I ask someone what happened, they say I was in editing mode but I wasn't

Hi there! I've just updated the release.
Do update here if problem still persists. Thanks!

~G

https://github.com/smrtgVirtual/hillgrove-city/releases/tag/v4.0.2-0810

**Comment by tib1247gg** [comment]
Still doesn't work. The AI still does not appear and the route hints still appears.

**Comment by smrtgVirtual** [comment]
> Still doesn't work. The AI still does not appear and the route hints still appears.

Thanks for the update. As a result I'll take this release off to investigate.

~G

**Comment by smrtgVirtual** [comment]
Hi!
Have again updated the files. It works fine on my side already.

Do tell me if it still persists

~G

https://github.com/smrtgVirtual/hillgrove-city/releases/tag/v4.0.2-0910
